bootstrap.py

```diff
@@ -285,11 +285,11 @@ def clean_environment():
 
 
 def generate_katello_facts():
-    if FQDN.find(".") != -1 and not os.path.exists('/etc/rhsm/facts/katello.facts'):
-        print_generic("Workaround for FQDN")
-        katellofacts = open('/etc/rhsm/facts/katello.facts', 'w')
-        katellofacts.write('{"network.hostname":"%s"}\n' % (FQDN))
-        katellofacts.close()
+    """Write katello_facts file based on FQDN"""
```

Comment:
Maybe add "should be run *after* katello-ca-consumer RPM was installed to override the generated facts there"?